docs/docs/tutorials/creating-fake-user-data.md

```diff
@@ -0,0 +1,257 @@
+
+## Elaborating a Data Model
+
+In the GA GUI, a target report would like like this:
+| User ID      | Sessions | Avg. Session Duration | Bounce rate | Revenue | Transactions | Goal Conversion Rate
+| ----------- | ----------- | ----------- | ----------- | ----------- | ----------- | ----------- |
+| fcnMA33      | 64       |  00:06:43 | 1.56% | $0.00 | 0 | 96.88% |
+| LAhfn66   | 48        | 00:07:59 | 2.08% | $0.00 | 0 | 33.33% |
+
+Before we start writing a code, let's look at the data closely. 
+
+In our case, in the table we see values in different formats. In reality though, we can limit them to just two data types: integers and floating numbers.
+
+Integer columns are:
+* Sessions
+* Transactions 
+
+Floating type columns are:
+* Bounce rate 
+* Avg. Session Duration
+* Revenue
+* Goal Conversion Rate
+
+For all columns, we need numbers that are equal or greater than zero. Besides, for the percentage columns, we need to introduce the upper limit. The duration column will be expressed in minutes since it is a bit more intuitive than hours and seconds. Additionally, we need to specify a reasonable maximum duration as well. 
+
+---
+**note**
+`revenue` and `transactions` have a dependency: a revenue can only be above zero if at least one transaction took place. \
+You will need to filter your data later and generate additional rows if not enough data has been left.
+
+---
+
+The same data in JSON format:
+
+```json
+{
+    "users":[
+        {
+        "user id": "fcnMA33",
+        "sessions": 64,
+        "avg session duration": 6.71,
+        "bounce rate": 1.56,
+        "revenue": 0.00,
+        "transactions": 0,
+        "goal conversion rate": 96.886
+        },
+        {
+        "user id": "LAhfn66",
+        "sessions": 48,
+        "avg session duration": 7.99,
+        "bounce rate": 2.08,
+        "revenue": 0.00,
+        "transactions": 0,
+        "goal conversion rate": 33.33
+        }
+    ]
+}
+```
+
+In the `synth` [number generators reference](https://www.getsynth.com/docs/content/number), you will find out that the tool offers one generator for both types of numbers but the output can be customized using different arguments and parameters.
+
+
+## Creating Namespace and Collection
+
+Similarly to inserting data into a database, we need to create a "bucket" first:
+* a [namespace](https://www.getsynth.com/docs/getting_started/core-concepts#namespaces) where you will store your data [schemas](https://www.getsynth.com/docs/getting_started/schema): a new directory on your storage drive
+* a [collection](https://www.getsynth.com/docs/getting_started/core-concepts#collections) for each schema: a JSON file
+
+In your CLI tool, execute the following commands:
+
+```commandline
+$ mkdir fake_ga
+```
+```commandline
+$ cd fake_ga && touch users.json
+```
+
+Now, we can start coding the data model.
+
+## Coding Your Data Model
+
+You need to open the `users.json` file in a code editor and reproduce the data model.
+
+```json
```

Comment:
Snippets of JSON that are valid Synth schemas can be tagged as `json synth` so that they are runnable in the rendered version.